src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_DeleteProfileScreen.tsx

```diff
@@ -33,9 +28,20 @@ export const Profile_DeleteProfileScreen = ({
     filterActiveOrCanBeUsedFareContracts(fareContracts, serverNow).length > 0;
 
   const [deleteError, setDeleteError] = useState<boolean>(false);
+  const {deleteCollectedData} = useBeaconsState();
+
+  const handleDeleteProfile = async () => {
+    const isProfileDeleted = await deleteProfile();
+    if (isProfileDeleted) {
+      await deleteCollectedData();
+      await signOut();
```

Comment:
Is the `navigation.navigate('Profile_RootScreen');` not needed anymore?

Reply:
No, it was removed due to an issue, signOut nows handles the redirection.